Resources/views/pjax_admin.html.twig

```diff
@@ -25,3 +25,19 @@
     </header>
     {% block content %}{% endblock %}
 </div>
+
+{% block notification %}
+{# Possible states are "error", "started", and "done" #}
```

Comment:
OK I will remove the comment.

> Maybe it could reference where the possible values should be looked for (and are maintained).

We currently don't have such place AFAIK.